### ISSUE TITLE
Windows TAP reset tutorial 

### ISSUE THREAD

**Issue by sailosha** [Jigsaw-Code/outline-client#368]
Sometimes if the local user changed its DNS or ipconfig setting, the outline will failure to run even though you uninstall and install it again. It happened when you didn't delete windows TAP correctly. Do we need add tutorials to tell how to reset TAP and re-install outline on the windows os? 

I seen this issue 3 times already.

**Comment by trevj** [comment]
@lgg Unfortunately, the tools bundled with tap-windows6 can only remove devices by their "hardware ID" which devices from other VPNs may share. We definitely do plan to fork/overhaul those tools at some point which would make this possible.

**Comment by trevj** [comment]
@sailosha I just realised that you're right, the installer currently does not try to "fix" the TAP device's subnet if the user has manually changed it. That would be straightforward and would help some users. Thanks for the suggestion.

**Comment by lgg** [comment]
@trevj sorry, I didn't really understand. 
>tap-windows6 can only remove devices by their "hardware ID" which devices from other VPNs may share

As I know: Outline Client creates TAP-adapter named 'outline-tap0' during installation, this TAP adapter is used only by Outline Client. 
In https://github.com/OpenVPN/tap-windows6 README: 
>`tapinstall remove TAP0901`

So I think that `tapinstall remove outline-tap0` should work fine in this case. Is it right?

**Comment by trevj** [comment]
@lgg Yes but if you also have, for example, the OpenVPN client installed then any network devices *it* has created will also be removed - as they share the same hardware ID.

Unfortunately, I don't know of a way to remove just outline-tap0 via the command line. Yes, this is slightly ridiculous.
